lib/plans/checkout.dart

```diff
@@ -473,4 +476,21 @@ class _CheckoutState extends State<Checkout>
       return false;
     }
   }
+
+
+  void hasPlansUpdateOrBuy() {
```

Comment:
As soon as the user generates a payment link, We start calling this method and compare it with old data to check if the user has made a payment or not.